wiki/Ranking_Criteria/id.md

```diff
@@ -0,0 +1,267 @@
+# Kriteria ranking
+
+*Untuk kriteria ranking spesifik [game mode](/wiki/Game_mode), lihat: [osu!](osu!), [osu!taiko](osu!taiko), [osu!catch](osu!catch), dan [osu!mania](osu!mania)*
```

Comment:
Inconsistency translation: `game mode` and `mode permainan`. I think it's ok to translate `game mode` to `mode permainan`. Change `Lihat` to `lihat` or `kunjungi`, en.md is using `see`, not `See`.

`*Untuk kriteria ranking [mode permainan](/wiki/Game_mode) secara spesifik, kunjungi: [osu!](osu!), [osu!taiko](osu!taiko), [osu!catch](osu!catch), dan [osu!mania](osu!mania)*`